spec/lib/web_of_science/harvester_spec.rb

```diff
@@ -136,4 +136,34 @@
       end
     end
   end
+
+  context 'when author query has too many publications' do
+    let(:lotsa_uids) { Array(1..Settings.WOS.max_publications_per_author) }
+    let(:query_author) { instance_double(WebOfScience::QueryAuthor) }
+
+    before do
+      allow(WebOfScience::QueryAuthor).to receive(:new).with(author, {}).and_return(query_author)
+      allow(query_author).to receive(:uids).and_return(lotsa_uids)
```

Comment:
These lines can roll into the instance_double

Reply:
done